Removes unwanted theme specific Column bottom margin

getdave opened the pull request:
Restores margin to original spacing value. Removes unwanted additional margin on last child.

## Description
Addresses https://github.com/WordPress/gutenberg/issues/14596

getdave committed:
Removes unwanted theme specific Column bottom margin

Restores margin to original spacing value. Removes unwanted additional margin on last child.

jasmussen commented:
Nice, this seems okay. Is there any overlap with #14407? Have you tested obscure combinations of blocks, such as paragraph, columns block with tons of inner paragraphs, then paragraphs after? how about images before and after columns block and images inside? If all's good then this PR is probably all good too.

getdave commented:
> Is there any overlap with #14407?

This PR only removes the superfluous line (that was added recently [here](https://github.com/WordPress/gutenberg/commit/c34d6b78e77cd4edbc2963003409151fb0939312#diff-d0807e2d5cf91a52ec75864e689a9d14)) that targeted the Theme directly. I think it was when I was testing against Twenty Nineteen. 

Should have no impact elsewhere.

jasmussen reviewed (approved):
Ship it.

m-e-h commented on packages/block-library/src/columns/style.scss:
```diff
@@ -10,6 +10,8 @@
 
 .wp-block-column {
+	margin-bottom: 1em;
+
```
If you do get a minute @getdave maybe just get rid of this blank line between properties?

getdave replied:
Done

getdave committed:
Removes unwanted whitespace in CSS

Addresses https://github.com/WordPress/gutenberg/pull/14614#discussion_r270152542

getdave commented:
Thanks @noisysocks. I was waiting on tests to pass which were taking ages to run. Appreciate your handling this.
